expr/bufferfilter.go

```diff
@@ -21,9 +22,9 @@ type BufferFilter struct {
 	op    int
 	left  *BufferFilter
 	right *BufferFilter
-	fnf   *fieldNameFinder
-	scf   *stringCaseFinder
-	sf    *stringFinder
+	fnf   *stringsearch.FieldNameFinder
+	scf   *stringsearch.CaseFinder
+	sf    *stringsearch.Finder
```

Comment:
This is fine for now, but I'd eventually like to rename these to `cf` and `f` to keep them in sync with their types.

Reply:
I'll change it.